controllers/sriovoperatorconfig_controller.go

```diff
@@ -78,6 +78,14 @@ func (r *SriovOperatorConfigReconciler) Reconcile(ctx context.Context, req ctrl.
 		Name: constants.DEFAULT_CONFIG_NAME, Namespace: namespace}, defaultConfig)
 	if err != nil {
 		if errors.IsNotFound(err) {
+			// Check if we only have one node
```

Comment:
This error won't be triggered unless user delete the default SriovOperatorConfig explicitly. Is it what we want? The default SriovOperatorConfig is created by OperatorConfig controller: https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/main.go#L154

Do we need to consider the upgrade scenario? e.g. from lower version to 4.10 which contains this fix. If not, I think adding the disableDrain setting in the above 154 line shall be sufficient.

Reply:
@zshi-redhat thanks! I miss that file!

I will like to also work on updates so I put the change in both places.

The reason I put it on both places is a was able to make a race from when I install the operator + a policy and until there is a reconcile to add the skip